pages/home/glossary.mdx

```diff
@@ -0,0 +1,158 @@
+
+### Authentication Provider
+
+An 'authentication provider' is a service that your users can authenticate to grant the agent access to their data or to take actions on their behalf.  This can be a hosted service provider like Google or Slack, or it can be a custom OAuth provider.  Multiple Toolkits may share the same authentication provider (e.g. Gmail and Google Drive share the same Google OAuth authentication provider).  Custom authentication providers are defined in the Arcade Dashboard or your Engine YAML if self-hosting.
+
+*Learn more about [authentication providers](/home/auth-providers).*
+
+### Authentication Scope
+
+An 'authentication scope' is a permission that a user can grant to an agent.  This is used to control what the agent can do with the user's data.  Available authentication scopes are defined by the authentication provider, and each tool defines the scopes it requires.
+
+Learn more about [authorized tool calling](/home/auth/auth-tool-calling).
+
+
+### Tool Executions
+
+A 'tool execution' is a single call to a tool to interact with a remote system or service. The tool execution itself may fail (e.g. the user does not have permission to call the tool), but as long as the execution was able to be routed to a worker, the tool execution will be counted.
+
+*Learn more about [tool executions](/home/use-tools/tools-overview).*
+
+
+### Standard and Pro Tool executions
+
+Arcade tools are divided into 2 categories: Standard and Pro.  While all tools have some cost for Arcade to run, Pro tools are significantly more costly - either due to infrastructure costs, the complexity of the tool, or a cost imposed by the provider of the tool.  Pro tools cost more to execute and have different limits.
+
+Learn more about tool pricing [here](https://www.arcade.dev/pricing).
+
+### Bring Your Own Credentials (BYOC)
+
+Bring Your Own Credentials (BYOC) is a feature that allows you to use your own credentials to certain pro tools.  This changes the cost of the tool execution, as you will be charged directly by the provider of the tool, rather than relying on Arcade to pay the bill for you.
+
+To set your own credentials, set the requisite secret within the Arcade Dashboard, overwriting the default 'static' credentials.
+
+
+### Tool Execution Flow
+
+```mermaid
+graph TD
+    A[Agent] --> B[Arcade Client]
+    B --> C[Arcade Engine]
+    C --> D[Worker]
+```
+
+### Arcade Client
+
+The 'Arcade client' is the SDK that you use to interact with the Arcade platform.  It is how your agent lists and calls tools, tied back to your project via an API key.  We offer clients for many popular languages and frameworks, including Python, JavaScript, and more.
+
+*Learn more about [the Arcade clients](/home/arcade-clients).*
```

Comment:
nit: need to define TDK / Tool Development Kit too! since that is separate from Client SDK now